src/components/offer-card/offer-actions/OfferActions.tsx

```diff
@@ -9,35 +10,25 @@ import TurnedInNot from '@mui/icons-material/TurnedInNot'
 
 import AppButton from '~/components/app-button/AppButton'
 
+import { authRoutes } from '~/router/constants/authRoutes'
+
 import { styles } from '~/components/offer-card/offer-actions/OfferActions.styles'
-import { ButtonActions, VariantEnum } from '~/types'
+import { VariantEnum } from '~/types'
 
 interface OfferActionsProps {
   id: string
   price: number
   isBookmarked: boolean
   onBookmarkClick: (id: string) => void
-  buttonActions: ButtonActions[]
 }
 
 const OfferActions: FC<OfferActionsProps> = ({
   id,
   price,
   isBookmarked,
-  onBookmarkClick,
-  buttonActions
+  onBookmarkClick
 }) => {
-  const buttons = buttonActions.map((elem, index) => (
```

Comment:
I think, you can do this without deleting this code, just change 'onclick' to 'to' and add component property to buttonActions